storybook/src/components/TextArea/TextArea.stories.tsx

```diff
@@ -20,22 +20,18 @@ const meta = {
     disabled: false,
   },
   argTypes: {
+    disabled: {
+      description: 'Prevents interaction. Use sparingly.',
+    },
     resize: {
       control: {
         type: 'radio',
         labels: { undefined: 'default', none: 'none', horizontal: 'horizontal', vertical: 'vertical' },
       },
       options: [undefined, 'none', 'horizontal', 'vertical'],
     },
-    cols: {
-      control: {
-        type: 'number',
-      },
-    },
-    rows: {
-      control: {
-        type: 'number',
-      },
```

Comment:
Why did you remove these? They show functionality we built into TextArea.

Reply:
True, I’ve restored them.